loops_cc.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+#Use a for loop to print the numbers 1 to 20, inclusive
+for value in range(1,20):
+    print(value)
+
+
+#Make a list of numbers from 1 to one million and then use a for loop to print the numbers
+numbers = list(range(1, 1000000))
+print(numbers)
+
+numbers = []
+for value in range(1, 1000000):
+    numbers.append(value)
+print(numbers)
+
+
+#Make a list of numbers from one to one million, then use min(), max(), and sum()
+numbers = [1, 1000000]
+print(min(numbers))
+print(max(numbers))
+print(sum(numbers))
+
```

Comment:
No need for an extra line here.

Reply:
@paulghaddad I add extra line breaks in this file b/c I compiled many short exercises together. The extra space separates the different exercises.